zeppelin-web/src/app/visualization/builtins/visualization-nvd3chart.js

```diff
@@ -81,6 +81,14 @@ export default class Nvd3ChartVisualization extends Visualization {
     return s;
   };
 
+  defaultY() {
```

Comment:
Shell we just `return 0` here and make `LinechartVisualization` override this method?